src/ast/mod.rs

```diff
@@ -4594,13 +4594,35 @@ impl fmt::Display for GrantObjects {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
 pub struct Assignment {
-    pub id: Vec<Ident>,
+    pub target: AssignmentTarget,
```

Comment:
I wonder if there is a reason to distinguish between a single column update and a multi column update?

I think it is common to update more than one column at a time in an UPDATE statement, and there is no reason to treat single column and multi-column updates differently

As in perhaps this could simply be

```suggestion
    pub target: Vec<ObjectName>,
```

Reply:
@alamb : We need a way to distinguish between `UPDATE t SET c = expr` and `UPDATE t SET (c) = expr`. The first one is the traditional UPDATE syntax (which supports multiple assignments, but that's not the question here), and the second one is the tuple update syntax, which expects `expr` to be a subselect or a row expression. The two are semantically different, so we need a way to represent them differently.

Reply:
Makes sense to me -- thank you for the clarification